src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -404,6 +405,19 @@ export default class CanvasController {
     }
 
     /**
+     * Fill entire scope with the background color specified in the current theme
+     *
+     * @for CanvasController
+     * @method canvas_fill_background
+     * @param cc {Object} HTML5 canvas context
+     */
+    canvas_fill_background(cc) {
+        cc.fillStyle = this.theme.SCOPE.BACKGROUND;
+
+        cc.fillRect(0, 0, this.canvas.size.width, this.canvas.size.height);
```

Comment:
Oh, okay, so actually it's because `this.canvas` is an actual reference to the actual canvas object. I thought it was a dummy property that contained a bunch of things that were updated every cycle (which would be an icky idea for sure).

So are you saying that you _would_ want to add and maintain `CanvasController.width` and `CanvasController.height` from the canvas? That would seem odd to me, but will drop the todo nonetheless if you would like.